src/main/java/mat/server/service/impl/MatContextServiceUtil.java

```diff
@@ -206,4 +233,13 @@ private BigDecimal asDecimalVersion(String version) {
         }
     }
 
+    public boolean isMeasureEditable(Measure measure) {
+        featureFlagMap = featureFlag.findFeatureFlags();
+        return featureFlagMap.get(FeatureFlagConstant.FHIR_EDIT) || MeasureDetailsUtil.QDM.equals(measure.getMeasureModel());

```

Comment:
getOrDefault(..., false) would be a better way to handle it

Reply:
Done